packages/core/admin/ee/server/controllers/authentication.js

```diff
@@ -62,4 +63,15 @@ module.exports = {
 
     return providerAuthenticationFlow(ctx, next);
   },
+
+  async isSSOLocked(ctx) {
```

Comment:
This probably makes more sense in the user controller than the auth controller, since it checks an account that is already authorized.